### DISCUSSION TITLE
Extended Scraping

### DISCUSSION THREAD

**Discussion by enigmalea** [FujoWebDev/AO3.js#38] [Ideas]
I thought it would be a good idea to document some of the things that aren't currently scraped by existing scrapers and therefore hasn't made it onto our lists. I'll start Comments for topics I'm aware of. Feel free to add to things as you wander through AO3.

**Comment by enigmalea** [comment]
**Collections**
- just in general I don't think there is a scraper that supports them. Would be nice to be able to get info on Collections.
- There is a TAGS page for collections that would be nice to scrape (shows you every tag that's on the works in the collection). It's as a tag cloud.

**Reply by essential-randomness** [reply]
Can you expand on the difference between Series and Collections?

**Reply by enigmalea** [reply]
Series are usually single author and are owned by one person. You can do collabs with them, but essentially ONLY the people who own the series can post to it. they're intended to be used for sequential stories meant to be read in an order.

Collections can be owned by a group of people and ANYONE can contribute (thought contributions can be moderated). They also have the ability to hold bookmarks of works. Collections are supposed to be used for works that fit a specific theme or event. Collections are where challenges are run.

I also use them to group fics about specific OCs together, fics with a certain tag, all my meta works with site/work skins. 

they're also just different sections of the AO3 architecture all together

**Reply by enigmalea** [reply]
BACK WITH VISUALS
[Series](https://archiveofourown.org/series/1285256)
[Collection w/ Works](https://archiveofourown.org/collections/dapolyex2022)
[Collection w/ Bookmarks](https://archiveofourown.org/collections/enigsrefs)

**Comment by enigmalea** [comment]
**Additional User Profile Pages**
- Bookmarks (like the works that have been bookmarked or rec'd)
- Gifts
- Works

**Reply by enigmalea** [reply]
to expand on my thoughts a little - it would be nice to be able to scrape your gifts to feature them on your website - or to be able to grab things you've identified as a rec to create a rec page.

**Reply by essential-randomness** [reply]
Does any of this require being logged in?

**Reply by enigmalea** [reply]
nope. no login required. you can have private bookmarks that would require login, but I think we can just document that any bookmarks that are private are inaccessible to the API.
